CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 # Change log
 All notable changes to this project will be documented in this file.
 
+## 0.16.0 - 2018-09-13
+### Summary:
+This release adds syslog_facility type, enhances syslog_facility and syslog_settings
+
+### Added:
+- `network_dns` type
```

Comment:
Yep should we highlight the type there rather than network_dns

it's nitpicky I know

`### Added:
- `network_dns`
  - hostname
- `syslog_facility` type
  - level
- `syslog_settings`
  - logfile_severity_level
  - logfile_name
  - logfile_size
  - buffered_severity_level
  - buffered_size`